islandora_solr_metadata.libraries.yml

```diff
@@ -0,0 +1,7 @@
+display:
+  js:
+    misc/form.js: {}
```

Comment:
File doesn't exist. Think we needed this from core in d7. Don't think we need it now.